docs/modules/ROOT/pages/index.adoc

```diff
@@ -32,6 +32,7 @@ Example (default) configuration for testing and local development:
 %dev,test.quarkus.wiremock.devservices.files-mapping=src/test/resources
 %dev,test.quarkus.wiremock.devservices.port=8089
 %dev,test.quarkus.wiremock.devservices.service-name=wiremock-server
+%dev,test.quarkus.wiremock.devservices.global-response-templating=true
```

Comment:
Currently, the examples are based on their default values. Maybe we could switch the `global-response-templating` attribute to `false`.